.gitlab-ci.yml

```
@@ -95,7 +95,7 @@ variables:
 
 .dind-login: &dind-login
   - mkdir -p $HOME/.docker && echo $DOCKER_AUTH_CONFIG > $HOME/.docker/config.json
-  - docker login --username $CI_REGISTRY_USER --password $CI_REGISTRY_PASSWORD $CI_REGISTRY
+  - for retry in {1..5}; do docker login --username $CI_REGISTRY_USER --password $CI_REGISTRY_PASSWORD $CI_REGISTRY && break || sleep $retry; done
```

Careful, this won't work on shell, it is a bash extension - we do use bash in here but I prefer to avoid these as we use to copy paste things around and soon or alter we mess it up :see_no_evil: 

Why not just:
```suggestion
  - for retry in 1 2 3 4 5; do docker login --username $CI_REGISTRY_USER --password $CI_REGISTRY_PASSWORD $CI_REGISTRY && break || sleep $retry; done
```

---

Smart suggestion! Thank you!